demo/00Index

```diff
@@ -1,2 +1,3 @@
 berlin	Demo for presentation at the 16th Berlin R meetup, July 2019
 zurich	Demo for presentation at the Zurich R meetup, September 2019
+demo Demo for presentation at Energie360, November 2019
```

Comment:
```suggestion
zurich-2 Demo for presentation at Energie360, November 2019
```

Reply:
To avoid failing R CMD check I would need to replace the entry for zurich.R in the index file, if  I overwrite zurich.R. Guess we can also sort this out then later